src/reader/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod crontab_reader;
+pub mod postgresql_reader;
```

Comment:
May I say that we need a DB abstraction ? I think we should not be highly coupled to PostgreSQL (despite the fact :cupid: PostgreSQL)